wiremock/deployment/src/main/java/io/quarkiverse/openapi/wiremock/generator/deployment/wiremock/model/Response.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkiverse.openapi.wiremock.generator.deployment.wiremock.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class Response {
```

Comment:
Can't this be a `record`?

Reply:
Hi @hbelmiro, yeah can be. Another point here is the use of `@JsonInclude` annotation, I do no know if makes sense to use it, I will try to remove it and use another way to not include non empty.